dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/AbstractImageRegistry.java

```diff
@@ -453,8 +460,8 @@ private void updateTags(List<Tag> newTags, @NotNull Tool tool, SourceCodeRepoInt
         }
 
         // For tools from dockerhub, grab/update the image and checksum information
-        if (tool.getRegistry().equals(Registry.DOCKER_HUB.toString())) {
-            updateDockerHubImageInformation(newTags, tool, existingTags);
+        if (tool.getRegistry().equals(Registry.DOCKER_HUB.toString()) || tool.getRegistry().equals(Registry.GITLAB.toString())) {
```

Comment:
If `tool.getRegistry()` ever returns null (?) you will get an NPE. If it's not possible, ignore this comment, otherwise reverse the comparisons, e.g., `Registry.DOCKER_HUB.toString().equals(tool.getRegistry())`. 

OTOH, you're following the existing pattern.

Reply:
Don't think it's possible. At least I don't see a null entry in the database.